projects/Mallard/src/constants/image-sizes.ts

```diff
@@ -0,0 +1,11 @@
+export const imageSizes = ['phone', 'tablet', 'tabletL', 'tabletXL'] as const
```

Comment:
should this just be moved from backend into common?

Reply:
Possibly - but the front end doesnt require common